apps/test/integration/util/testCollectionUtils.js

```diff
@@ -1,17 +1,43 @@
+/* global LEVEL_TYPE */
+
 var _ = require('lodash');
 
+/**
+ * Given a list of levelSolutions subdirectories, returns only those matching
+ * the pattern specified in LEVEL_TYPE. If specified, LEVEL_TYPE must be contain
+ * one or more subdirectory names joined with '|' e.g. 'applab' or
+ * 'maze|artist|craft'.
```

Comment:
an `npm test:help` would be nice. For now I've added docs for this new env variable in apps/README.md.